src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -823,9 +823,12 @@ module Extensions =
             { current with ConnectionStrings = current.ConnectionStrings.Add(key, (ParameterSetting (SecureParameter key), Custom)) }
             |> this.Wrap state
         member this.AddConnectionString(state:'T, (key, value:ArmExpression)) =
+            this.AddConnectionString(state, (key, value, Custom))
+        member this.AddConnectionString(state:'T, (key, value:ArmExpression, kind)) =
             let current = this.Get state
-            { current with ConnectionStrings = current.ConnectionStrings.Add(key, (ExpressionSetting value, Custom)) }
+            { current with ConnectionStrings = current.ConnectionStrings.Add(key, (ExpressionSetting value, kind)) }
             |> this.Wrap state
+            
         /// Creates a set of connection strings of the web app whose values will be supplied as secret parameters.
         [<CustomOperation "connection_strings">]
         member this.AddConnectionStrings(state:'T, connectionStrings) =
```

Comment:
Could you try to add an overload here as well, so that people could add multiple connection strings and still specify the connection type?

Reply:
Is this the change you have in mind?
``` fsharp
        /// Creates a set of connection strings of the web app whose values will be supplied as secret parameters.
        [<CustomOperation "connection_strings">]
        member this.AddConnectionStrings(state:'T, connectionStrings) =
            this.AddConnectionStrings(state, connectionStrings, Custom)
        member this.AddConnectionStrings(state:'T, connectionStrings, kind) =
            let current = this.Get state
            connectionStrings
            |> List.fold (fun (state:CommonWebConfig) (key, value:ArmExpression) ->
               { state with ConnectionStrings = state.ConnectionStrings.Add(key, (ExpressionSetting value, kind)) }) current 
            |> this.Wrap state
```

Adding a new test
``` fsharp
    test "Web App correctly adds mutliple connection strings in bulk" {
        let wa =
            let resources = webApp { name "test"
                                     connection_strings [
                                         ("a", ArmExpression.create("a"))
                                         ("b", ArmExpression.create("b"))
                                     ], SQLAzure
                                     } |> getResources
            resources |> getResource<Web.Site> |> List.head

        let expected = [
            "a", (ExpressionSetting (ArmExpression.create("a"), SQLAzure)
            "b", (ExpressionSetting (ArmExpression.create("b")), SQLAzure)
        ]
        let parameters = wa :> IParameters

        Expect.equal wa.ConnectionStrings (Map expected |> Some) "Missing connections"
    }
```
Gives me this: `WebApp.fs(92, 9): [FS0010] Unexpected symbol ']' in expression`. (Closing square bracket for `let expected`). What syntax should I be using here?